src/views/virtualmachines/actions/hooks/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { MigPlan } from '@kubevirt-utils/resources/migrations/constants';
+
+export const sortMigPlansByCreationTimestamp = (a: MigPlan, b: MigPlan): number =>
+  a?.metadata?.creationTimestamp?.localeCompare(b?.metadata?.creationTimestamp);
```

Comment:
the comparator is not symetric  - it should treat the falsy values the same on both sides. Currently it's not the case:
```
["a","","b",undefined,"c", null].sort((a,b)=> a?.localeCompare(b))
Array(6) [ "", "a", "b", "c", null, undefined ]
```

Reply:
oh i understand but the creationTimestamp will not be `""`. Maybe `a` or `b` are undefined so I guarded that with `?`